### DISCUSSION TITLE
Migrate DB

### DISCUSSION THREAD

**Discussion by andrey-helldar** [HassanZahirnia/laravel-package-ocean#23] [Package Suggestions]
Easy data transfer from one database to another

https://github.com/TheDragonCode/migrate-db

**Comment by HassanZahirnia** [comment]
Hey, this package already exists in the database! Please re-check your suggestions in the live site before suggesting them.

Thanks anyway!

**Reply by andrey-helldar** [reply]
Sorry, I don't remember adding the package to this site, that's why I sent it 🙂
